### PR TITLE
Add Disable SSL and Force Path Style options.

### PR DESCRIPTION
**Pull request by maxpospelov**
Hello, I had a task to deploy the S3 exporter and integrate with our local minio service.
I have added new variable for that.
Thank you for your work and maybe my little update will be helpful for you too.

### CODE REVIEW

File: s3_exporter.go [hunk 1]
```diff
@@ -149,6 +149,8 @@ func main() {
 		metricsPath   = app.Flag("web.metrics-path", "Path under which to expose metrics").Default("/metrics").String()
 		probePath     = app.Flag("web.probe-path", "Path under which to expose the probe endpoint").Default("/probe").String()
 		endpointURL   = app.Flag("s3.endpoint-url", "Custom endpoint URL").Default("").String()
+		disableSSL    = app.Flag("s3.disable-ssl", "Custom disable SSL").Bool()
+		forcePathStyle = app.Flag("s3.force-path-style", "Custom force path style").Bool()
```
Comment (ribbybibby) [inline s3_exporter.go:153]: Can you align these `=` signs, please?

**Review by ribbybibby — CHANGES_REQUESTED** [review]
Can you please document these options in the README?

File: s3_exporter.go (hunk 1 above)
Comment (maxpospelov) [reply s3_exporter.go:153]: Done

**Comment by maxpospelov** [comment]
> Can you please document these options in the README?

I documented these options in the  README.

**Comment by maxpospelov** [comment]
@ribbybibby Can you check it please?

**Review by ribbybibby — APPROVED** [review]
Thanks @maxpospelov. Sorry for the long wait; it looks good to me.

**Comment by snqk** [comment]
@ribbybibby could you make a release & update dockerhub for this commit please?

**Pull request merged**